static/app/views/settings/account/accountSecurity/sessionHistory/index.spec.tsx

```diff
@@ -39,8 +39,8 @@ describe('AccountSecuritySessionHistory', function () {
 
     render(<SessionHistory {...routerProps} />, {context: RouterContextFixture()});
 
-    expect(screen.getByText('127.0.0.1')).toBeInTheDocument();
-    expect(screen.getByText('192.168.0.1')).toBeInTheDocument();
-    expect(screen.getByText('US (CA)')).toBeInTheDocument();
+    expect(await screen.findByText('127.0.0.1')).toBeInTheDocument();
```

Comment:
tiny nit but you should only need to use findBy on the first one, and then since things are already loaded after that can be found using getBy